lib/commons/color/get-background-color.js

```diff
@@ -65,9 +77,48 @@ function calculateObscuringAlpha(elmIndex, elmStack) {
 }
 
 function elmOutsideParent(elm, bgElm, bgColor) {
```

Comment:
I'm going to keep the logic in this function the same–it doesn't actually impact the case I was trying to address (inline labels triggering `bgOverlap`), and I don't want to regress other cases this was intended to fix. I'll rename it to `elmPartiallyObscured` so the missingData exposed is less confusing.